django_select2/forms.py

```diff
@@ -108,6 +108,21 @@ def _get_media(self):
 
     media = property(_get_media)
 
+    def label_from_instance(self, obj):
+        """
+        Label representation from instance.
+
```

Comment:
Make sure to drop a line that this will be used to create the select options. Focus on these to words, as they are what people will be looking fore, since they come from an html world and know nothing about this code.